strr-api/src/strr_api/responses/RegistrationSerializer.py

```diff
@@ -91,6 +91,9 @@ def _populate_header_data(cls, registration_data: dict, registration: Registrati
         if application:
             registration_data["header"]["applicationNumber"] = application.application_number
             registration_data["header"]["applicationDateTime"] = application.application_date.isoformat()
+            registration_data["header"]["organizationName"] = (
+                application.application_json.get("registration").get("strRequirements", {}).get("organizationNm")
```

Comment:
Just to clarify, the `organizationNm` is always present? For all applications?

Reply:
No, it is not always present, `header-> organizationName` will be None in those cases.